README.md

```diff
@@ -45,42 +45,46 @@ A wide variety of examples showcasing what Nutrient Web SDK can do can be found
+- [Restrict dragging and editing ink annotations and signatures](https://nutrient.io/playground?p=eyJ2IjoxLCJzZXR0aW5ncyI6eyJmaWxlTmFtZSI6ImZvcm0ucGRmIiwiand0Ijp7InBlcm1pc3Npb25zIjpbInJlYWQtZG9jdW1lbnQiLCJ3cml0ZSJdLCJ1c2VySWQiOiJyYW5kb211cWlpT202ZU5xIiwibGF5ZXIiOiJyYW5kb21QRzRBRkxwbGM2In19LCJqcyI6IlBTUERGS2l0LmxvYWQoey4uLmJhc2VPcHRpb25zLHRoZW1lOlBTUERGS2l0LlRoZW1lLkRBUkt9KS50aGVuKChhc3luYyBpbnN0YW5jZT0%252Be2NvbnN0IGFubm90YXRpb249bmV3IFBTUERGS2l0LkFubm90YXRpb25zLklua0Fubm90YXRpb24oe3BhZ2VJbmRleDowLGlzU2lnbmF0dXJlOiEwLGxpbmVzOlBTUERGS2l0LkltbXV0YWJsZS5MaXN0KFtQU1BERktpdC5JbW11dGFibGUuTGlzdChbbmV3IFBTUERGS2l0Lkdlb21ldHJ5LkRyYXdpbmdQb2ludCh7eDo1LHk6NX0pLG5ldyBQU1BERktpdC5HZW9tZXRyeS5EcmF3aW5nUG9pbnQoe3g6OTUseTo5NX0pXSksUFNQREZLaXQuSW1tdXRhYmxlLkxpc3QoW25ldyBQU1BERktpdC5HZW9tZXRyeS5EcmF3aW5nUG9pbnQoe3g6OTUseTo1fSksbmV3IFBTUERGS2l0Lkdlb21ldHJ5LkRyYXdpbmdQb2ludCh7eDo1LHk6OTV9KV0pXSksYm91bmRpbmdCb3g6bmV3IFBTUERGS2l0Lkdlb21ldHJ5LlJlY3Qoe2xlZnQ6MCx0b3A6MCx3aWR0aDoxMDAsaGVpZ2h0OjEwMH0pfSk7YXdhaXQgaW5zdGFuY2UuY3JlYXRlKGFubm90YXRpb24pLGluc3RhbmNlLmFkZEV2ZW50TGlzdGVuZXIoJ2Fubm90YXRpb25zLnByZXNzJywoZXZlbnQ9Pntjb25zdHthbm5vdGF0aW9uOmFubm90YXRpb24sbmF0aXZlRXZlbnQ6bmF0aXZlRXZlbnR9PWV2ZW50O2NvbnNvbGUubG9nKGluc3RhbmNlLmdldFNlbGVjdGVkQW5ub3RhdGlvbnMoKSksbnVsbCE9PWluc3RhbmNlLmdldFNlbGVjdGVkQW5ub3RhdGlvbnMoKSYmYW5ub3RhdGlvbiBpbnN0YW5jZW9mIFBTUERGS2l0LkFubm90YXRpb25zLklua0Fubm90YXRpb24mJmFubm90YXRpb24uaXNTaWduYXR1cmUmJihldmVudC5wcmV2ZW50RGVmYXVsdCgpLGluc3RhbmNlLnNldFNlbGVjdGVkQW5ub3RhdGlvbnMoYW5ub3RhdGlvbikpfSkpfSkpOyIsImNzcyI6Ii8qIEFkZCB5b3VyIENTUyBoZXJlICovXG5cbi5QU1BERktpdC1SZXNpemUtQW5jaG9yIHtcbiAgICBkaXNwbGF5OiBub25lO1xufVxuXHQifQ%253D%253D)

```

Comment:
Same.

Reply:
I have no idea about intentions etc. I just updated the text to describe the example